s4e-backend/src/main/java/pl/cyfronet/s4e/service/InstitutionService.java

```diff
@@ -139,8 +139,14 @@ public void update(UpdateInstitutionRequest request, String institutionSlug)
             fileStorage.delete(getEmblemKey(institutionSlug));
         }
         uploadEmblemIfRequested(slug, request.getEmblem());
+
         institution.setName(request.getName());
         institution.setSlug(slug);
+        institution.setAddress(request.getAddress());
+        institution.setCity(request.getCity());
+        institution.setPostalCode(request.getPostalCode());
+        institution.setPhone(request.getPhone());
+        institution.setSecondaryPhone(request.getSecondaryPhone());
         update(institution);
```

Comment:
I don't think this is necessary, this call should be removed alongside the method calling save.
You\re within a transaction here, so any changes will be saved automatically when leaving it.